crates/ruff_python_formatter/src/statement/stmt_expr.rs

```diff
@@ -10,6 +11,14 @@ impl FormatNodeRule<StmtExpr> for FormatStmtExpr {
     fn fmt_fields(&self, item: &StmtExpr, f: &mut PyFormatter) -> FormatResult<()> {
         let StmtExpr { value, .. } = item;
 
+        if let Some(constant) = value.as_constant_expr() {
```

Comment:
It seems that `StmtExpr` is the only one where we don't want to break string continuations. So I added an explicit exclude.

Reply:
Using implicit concatenations on statement level feels strange to me, do we have any examples of that in the wild